base/PerfOptions.php

```diff
@@ -235,7 +235,7 @@ public function __construct(Vector<string> $argv) {
         $this->tcprint = $tcprint;
       } else if ($isFacebook) {
         $this->tcprint =
-          $fbcode . '/_bin/hphp/facebook/tools/tc-print/tc-print';
+          $fbcode . '/_bin/hphp/tools/tc-print/tc-print';
```

Comment:
The rest of that path is still facebook specific, our cmake build doesn't create an _bin directory. A flag could be added to search for things in the right place in a cmake build, I just haven't added it.